test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -263,7 +263,23 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 			// defaultCpuNotInSchedulingDomains is empty if no gu pods are running
 			defaultCpuNotInSchedulingDomains, err := getCPUswithLoadBalanceDisabled()
 			Expect(err).ToNot(HaveOccurred(), "Unable to fetch scheduling domains")
-			testlog.Infof("Default scheduling Domains are: %s", defaultCpuNotInSchedulingDomains)
+
+			if len(defaultCpuNotInSchedulingDomains) > 0 {
+				pods, err := pods.GetPodsOnNode(workrtRTNode.Name)
+				if err != nil {
+					testlog.Warningf("cannot list pods on %q: %v", workerRTNode.Name, err)
+				} else {
+					testlog.Infof("pods on %q BEGIN", workerRTNode.Name)
+					for _, pod := range pods {
+						testlog.Infof("- %s/%s %s annotations=%#v", pod.Namespace, pod.Name, pod.UID, pod.Annotations)
+					}
+					testlog.Infof("pods on %q END", workerRTNode.Name)
+				}
+
+				// TODO: should this be a skip?
```

Comment:
The base assumption is that no pods with lb disabled are in the cluster in the beginning of the test, hence we should Fail here.

Reply:
addressed below. Thanks!